bundles/org.openhab.binding.solarforecast/src/main/resources/OH-INF/config/fs-plane-config.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:solarforecast:fs-plane">
+		<parameter name="refreshInterval" type="integer" min="1" unit="min" required="true">
+			<label>Forecast Refresh Interval</label>
+			<description>Data refresh rate of forecast data</description>
```

Comment:
Since the unit declared above is not visible in the UI, this might help the users:
```suggestion
			<description>Data refresh rate of forecast data (in minutes)</description>
```

Reply:
Done